packages/atlas/src/views/global/YppLandingView/YppAuthorizationModal/YppAuthorizationModal.tsx

```diff
@@ -163,11 +170,13 @@ export const YppAuthorizationModal: FC<YppAuthorizationModalProps> = ({
       ...(selectedChannelId ? { joystreamChannelId: parseInt(selectedChannelId) } : {}),
       authorizationCode: ytResponseData?.authorizationCode,
       userId: ytResponseData?.userId,
-      videoCategoryId: data.videoCategoryId,
       email: data.email,
+      // todo: uncomment when https://github.com/Joystream/youtube-synch/issues/105 is done
+      // isSyncActive: data.isSyncActive,
```

Comment:
We can uncomment this already. Also, I think this will be called `shouldBeIngested`